pkg/operation/seed/seed.go

```diff
@@ -1293,3 +1311,26 @@ func deleteIngressController(ctx context.Context, c client.Client) error {
 		&corev1.Service{ObjectMeta: metav1.ObjectMeta{Name: "nginx-ingress-k8s-backend", Namespace: v1beta1constants.GardenNamespace}},
 	)
 }
+
+func comparePriorityClassValue(ctx context.Context, k8sClient client.Client, priorityClassName string, valueToCompare int) (int, error) {
+	pc := &schedulingv1.PriorityClass{}
+	if err := k8sClient.Get(ctx, kutil.Key(priorityClassName), pc); client.IgnoreNotFound(err) != nil && !meta.IsNoMatchError(err) {
+		return 0, err
+	}
+	return valueToCompare - int(pc.Value), nil
```

Comment:
if the resource does not exist, this `valueToCompare` what value will have?

Reply:
`valueToCompare ` is passed as argument it does not depends on any resource

Reply:
Sorry. I mean `pc.Value`. It should be zero?